### PR TITLE
added bootstrap-4

### PR DESCRIPTION
**Pull request by DinhManhHoang**

### CODE REVIEW

File: manhhoang/bootstrap/contactus.html [hunk 1]
```diff
@@ -9,9 +9,9 @@
          content must come *after* these tags -->
     <title>Ristorante Con Fusion: About Us</title>
         <!-- Bootstrap -->
-    <link href="css/bootstrap.min.css" rel="stylesheet">
-    <link href="css/bootstrap-theme.min.css" rel="stylesheet">
-    <link href="css/font-awesome.min.css" rel="stylesheet">
+    <link href="bower_components/bootstrap/dist/css/bootstrap.min.css" rel="stylesheet">
+    <link href="bower_components/bootstrap/dist/css/bootstrap-theme.min.css" rel="stylesheet">
```
Comment (superpoor) [inline manhhoang/bootstrap/contactus.html]: 2 spaces tab nhé e

File: manhhoang/bootstrap/css/mystyle.css [hunk 2]
```diff
@@ -1,87 +1,96 @@
-.row-header{
-    margin:0px auto;
-    padding:0px auto;
+.row-header {
+  margin: 0px auto;
+  padding: 0px 0px;
 }
-
 .row-content {
-    margin:0px auto;
```
Comment (superpoor) [inline manhhoang/bootstrap/css/mystyle.css]: 2 spaces tab

File: manhhoang/bootstrap/css/mystyle.css [hunk 3]
```diff
@@ -1,87 +1,96 @@
...
-
-.navbar-inverse .navbar-nav .open .dropdown-menu> li> a:hover {
-    color:#000000;
+.navbar-inverse .navbar-nav > .open > a {
+  background: #1a237e;
+  color: #ffffff;
 }
-
-.jumbotron {
-    padding: 70px 30px 70px 30px;
```
Comment (superpoor) [inline manhhoang/bootstrap/css/mystyle.css]: 2 spaces

File: manhhoang/bootstrap/css/mystyle.less [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+@off-white: #eeeeee;
+@lt-gray: #dddddd;
+@indigo: #303f9f;
+@dark-indigo: #1a237e;
+@light-indigo: #7986CB;
+
+@carousel-height: 300px;
+
+.zero-margin (@pad-up-dn: 0px, @pad-left-right: 0px) {
+	margin: 0px auto;
```
Comment (superpoor) [inline manhhoang/bootstrap/css/mystyle.less]: 2 spaces

File: manhhoang/bootstrap/index.html [hunk 5]
```diff
@@ -330,8 +334,25 @@ <h2 class="media-heading">Alberto Somayya</h2>
         </div>
     </footer>
     <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
-    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+    <script src="bower_components/jquery/dist/jquery.min.js"></script>
     <!-- Include all compiled plugins (below), or include individual files as needed -->
-    <script src="js/bootstrap.min.js"></script>
+    <script src="bower_components/bootstrap/dist/js/bootstrap.min.js"></script>
+    <script>
+    		$(document).ready(function(){
```
Comment (superpoor) [inline manhhoang/bootstrap/index.html]: 2 spaces

**Comment by DinhManhHoang** [comment]
tab fixed

**Pull request merged**